<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QWidget" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>344</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>344</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="sum_credit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> rub</string>
   </property>
   <property name="maximum">
    <double>99999999999.990005493164063</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="term">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="procent">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> %</string>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>9999999.990000000223517</double>
   </property>
  </widget>
  <widget class="QComboBox" name="year_month">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>56</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Месцев</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="type_credit_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип ежемесячного</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="differentiated">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированные</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="annuity">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>108</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетные</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="type_credit_label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>платежа</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="term_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="procent_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthly_payment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячный платёж</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_charges">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="debt_interest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Долг + проценты</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthly_payment_num">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_charges_num">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="debt_interest_num">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_calc">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>200</y>
     <width>336</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
